datasetinsights/stats/visualization/plots.py

```diff
@@ -5,14 +5,17 @@
 import pandas as pd
 import plotly.express as px
 import plotly.graph_objects as go
-from PIL import Image, ImageColor
+from PIL import Image, ImageColor, ImageDraw
 
 from datasetinsights.stats.visualization.bbox2d_plot import (
     add_single_bbox_on_image,
 )
 from datasetinsights.stats.visualization.bbox3d_plot import (
     add_single_bbox3d_on_image,
 )
+from datasetinsights.stats.visualization.keypoints_plot import (
```

Comment:
Can simplify this to `from .keypoints_plot import draw_keypoints_for_figure`

https://docs.python.org/3/reference/import.html#package-relative-imports